cmd/kyverno/main.go

```diff
@@ -111,14 +111,12 @@ func main() {
 		os.Exit(1)
 	}
 
-	// ======================= resource cache ====================
 	rCache, err := resourcecache.NewResourceCache(log.Log, clientConfig, client, []string{"configmaps"}, []string{})
 	if err != nil {
-		setupLog.Error(err, "Failed to create resource cache")
-		os.Exit(1)
+		setupLog.Error(err, "ConfigMap lookup disabled: failed to create resource cache")
```

Comment:
Lets start all messages with lower case. Here is a suggestion:

   `failed to create ConfigMap cache`

Reply:
And what is the recovery from this?

Reply:
The initial design was to support any kind of resource cache, so it used a shared informer so the informers can be registered dynamically, from there it queried the preferred API version and group.  Since we constantly seeing an issue with group `external.metrics.k8s.io`, I now change to only register informer for configMap (the only Kind we support for data lookup), it should not return any error now.

Once we add support for API server data lookup, we can decide do we still want to keep this or remove it completely.